PowerEditor/src/Parameters.h

```diff
@@ -296,7 +300,8 @@ struct CmdLineParamsDTO
 		dto._pos2go = params._pos2go;
 		
 		dto._langType = params._langType;
-		dto._udlName = params._udlName;
+		wcsncpy(dto._udlName, params._udlName.c_str(), MAX_PATH);
+		wcsncpy(dto._pluginMessage, params._pluginMessage.c_str(), MAX_PATH);
 		return dto;

```

Comment:
The `wscncpy(...)` will ensure the serialization of incoming string pointer data.